hashmail_server.go

```diff
@@ -568,16 +574,16 @@ func (h *hashMailServer) NewCipherBox(ctx context.Context,
 		return nil, err
 	}
 
-	log.Debugf("New HashMail stream init: id=%x, auth=%v",
-		init.Desc.StreamId, init.Auth)
+	ctxl := btclog.WithCtx(ctx, btclog.Hex("stream_id", init.Desc.StreamId))
 
-	if err := h.ValidateStreamAuth(ctx, init); err != nil {
-		log.Debugf("Stream creation validation failed (id=%x): %v",
-			init.Desc.StreamId, err)
+	log.DebugS(ctxl, "New HashMail stream init", "auth", init.Auth)
+
+	if err := h.ValidateStreamAuth(ctxl, init); err != nil {
+		log.DebugS(ctxl, "Stream creation validation failed", err)
```

Comment:
I think this needs to be `"err", err`, otherwise this would show as "bad key", because `DebugS` doesn't have an `err` parameter. Same further below.

Reply:
a yes good catch! 

makes me wonder if we should switch this to a "WarnS"? but anyways, keeping it as debug for now 👍